tests/libnet/ipaddress.go

```diff
@@ -1,9 +1,13 @@
 package libnet
 
 import (
+	"net"
+
 	k8sv1 "k8s.io/api/core/v1"
 	netutils "k8s.io/utils/net"
 
```

Comment:
I think we can join the ptpip import to the k8s ones. or do you also separate k8s org related imports?

Reply:
I saw that in other places, for example :
https://github.com/kubevirt/kubevirt/blob/master/tests/vmi_multus_test.go#L27
the go packages  are kept separate from the k8s packages, so I did the same.

Do you think that because there are not so many imports, I should join them?

Reply:
no. I saw this done as well. not sure about it, that's why I asked :)